Procfile_docker

```diff
@@ -0,0 +1,4 @@
+web: bundle exec puma -C config/puma.rb -e development -p 3001
+webpacker: ./bin/webpack-dev-server # Live update webpack js
+log: tail -f log/development.log
+hard_worker: bundle exec rerun --background --dir app,db,lib --pattern '{**/*.rb}' --no-notify -- bundle exec sidekiq
```

Comment:
Is there a reason that you are using a `Procfile_docker` rather than the development version, since they're the same?

Reply:
They are almost the same, on the Docker version I added the `--no-notify` option.

I wrote here more details on what I found out.
https://github.com/bikeindex/bike_index/pull/1562#pullrequestreview-398024453

Reply:
Ahh, ok, thanks for the explanation!